controller/internal/enforcer/nfqdatapath/datapath.go

```diff
@@ -48,6 +49,8 @@ var collectCounterInterval = 30 * time.Second
 // GetUDPRawSocket is placeholder for createSocket function. It is useful to mock tcp unit tests.
 var GetUDPRawSocket = afinetrawsocket.CreateSocket
 
+var ackSize uint32 = 136 + uint32(len(tokens.AckPattern))
```

Comment:
if 136 is a magic number we introduced there should be a comment saying what does this signify